server/routes/temporary-accommodation/manage.ts

```diff
@@ -252,7 +252,7 @@ export default function routes(controllers: Controllers, services: Services, rou
     ],
   })
 
-  get(paths.reports.new.pattern, reportsController.new(), { auditEvent: 'VIEW_REPORT_CREATE' })
+  get(paths.reports.index(), reportsController.index(), { auditEvent: 'VIEW_REPORT_INDEX' })
```

Comment:
We still need a reference to the current `paths.reports.new()` here, otherwise it won't be handled at all -- in fact this may be a better place to hold the redirect itself, with a new `redirectsController` maybe? This would remove the need to add the redirect logic everywhere it is needed.

```suggestion
  // Redirect using a generic redirects controller, with redirect method taking location and optional status code
  get(paths.reports.new(), redirectsController.redirect(paths.reports.index(), 301), { auditEvent: 'REDIRECT_REPORT_NEW' })
  get(paths.reports.index(), reportsController.index(), { auditEvent: 'VIEW_REPORT_INDEX' })
```

Reply:
Love this suggestion, I was following the existing pattern that was used for Referralsl. I'll update referrals as well so theres no redirects happening in the controllers